### PR TITLE
build: updated some warning strings; properly refresh lua files embedded in falco

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> If contributing rules or changes to rules, please make sure to also uncomment one of the following line:

> /kind rule-update

> /kind rule-create

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area build

/area engine

> /area rules

> /area tests

> /area proposals

**What this PR does / why we need it**:

* Any load_rules warning is now printed (not only on verbose mode)
* Added back a "performance hit" comment when ruleset has a rule without evt.type
* Check for ignored events warning only when "-A" flag is not actually passed
* Fixed a lua CMakeLists issue that prevented updated lua files to be refreshed in falco

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: userspace/engine/lua/CMakeLists.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@
 # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
-file(GLOB_RECURSE lua_module_files ${CMAKE_CURRENT_SOURCE_DIR} *.lua)
+file(GLOB_RECURSE lua_files ${CMAKE_CURRENT_SOURCE_DIR} *.lua)
```
Comment (leogr) [inline userspace/engine/lua/CMakeLists.txt:13]: :star_struck: 

Thank you :pray:

**Review by leogr — COMMENTED** [review]
Thank you @FedeDP for fixing these last-minute issues.
SGTM, I have just left some suggestions.

**Review by leogr — APPROVED** [review]
LGTM now

Thank you

/approve

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**